columbo/_interaction.py

```diff
@@ -481,6 +482,15 @@ def get_answers(
     return result
 
 
+def canonical_arg_name(name: str) -> str:
+    sanizized_name = (
+        f"{name.lower().replace(' ', '-').replace('_', '-').lstrip('-').rstrip('-')}"
```

Comment:
Does this need to be an f-string?

Reply:
+1. Also, `.strip()` can be used in place of `.lstrip().rstrip()`